content/en/developers/metrics/custom_metrics.md

```diff
@@ -3,226 +3,244 @@ title: Custom Metrics
+There are less custom metrics if only a subset of services and statuses are reporting. For example, you have three hosts:
 
-From there, you can see that **on each host reporting this metric**, if all services report both successes and failures, you can have up to 1x2x3 = **6 custom metrics**.
+* `host1` is reporting all possible configurations.
+* `host2` is reporting only successes across all services.
+* `host3` is reporting successes and failures for database and webserver services.
 
-Let's say you have 3 hosts:
-
-* `host1` is reporting all possible configurations
-* `host2` is reporting only successes across all services
-* `host3` is reporting success and failures, but only for database and webserver services
-
-Across your 3 hosts, you'd have 13 distinct metrics, here is why :
+Across your three hosts, there are 13 distinct metrics:
 
 {{< img src="developers/metrics/custom_metrics/metric_count.png" alt="metric_count" responsive="true" style="width:75%;">}}
 
-If you are an administrator, you can see your total custom metrics per hour as well as the top 500 custom metrics by cardinality in your account in [the usage details page][7]. You can also see this metric count on your [metric summary page][8], where you'd see, clicking on the service.request.count metric, the exact number of unique tag combinations:
+#### Metric summary
+
+Your [metric summary page][9] shows the number of distinct metrics for a specific metric. For example, `service.request.count` with 1 host, 2 statuses, and 3 services = **6 distinct metrics**:
 
-So if you only had the first host from the example above reporting, you'd have this:
+{{< img src="developers/metrics/custom_metrics/metric_summary.png" alt="metric_summary" responsive="true" style="width:80%;">}}
 
-{{< img src="developers/metrics/custom_metrics/metric_summary.png" alt="metric_summary" responsive="true" style="width:70%;">}}
+Adding a second host with 3 services and 1 status = **9 distinct metrics**:
 
-Adding the second host:
+{{< img src="developers/metrics/custom_metrics/metric_summary_2.png" alt="metric_summary_2" responsive="true" style="width:80%;">}}
 
-{{< img src="developers/metrics/custom_metrics/metric_summary_2.png" alt="metric_summary_2" responsive="true" style="width:70%;">}}
+Adding a third host with 2 services and 2 status = **13 distinct metrics**:
 
-Adding the third host as per the table above, you get your 13 distinct metrics:
+{{< img src="developers/metrics/custom_metrics/metric_summary_3.png" alt="metric_summary_3" responsive="true" style="width:80%;">}}
 
-{{< img src="developers/metrics/custom_metrics/metric_summary_3.png" alt="metric_summary_3" responsive="true" style="width:70%;">}}
+#### Query editor
 
-Using the query editor, you can also find this using the count: aggregator
+You can count your custom metrics by using the `count:` aggregator in the query editor. For counting the previous example, the query `count:service.request.count{*}` is used:
```

Comment:
:mindblown: I didn't know the `count:` aggregator existed! Do users have to edit the query's JSON directly to use it? (I don't see it as a regular aggregator in the UI, like `sum`, `avg`, etc)

(if so, should we mention it here? It's a bit of a hidden aggregator)

Reply:
Indeed they have to edit the JSON directly in order to benefit from it. Idk why it's hidden in the UI i'm going to investigate and update this into another PR if need be.